CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [27.1.0]
+### Added
+- Now the `KeyringController` supports the `QRKeyring` from `@keystonehq/metamask-airgapped-keyring`. Developers can enable the import of accounts from a QR hardware wallet. A new optional parameter, `setAccountLabel` from the `PreferencesController`, should be passed to the `KeyringController` to enable this new functionality. ([#685](https://github.com/MetaMask/controllers/pull/685))
+
+### Changed
+- Bump `eth-phishing-detect` version from 1.1.14 to 1.1.16 ([#742](https://github.com/MetaMask/controllers/pull/742))
+- Bump `@metamask/contract-metadata` from 1.31.0 to 1.33.0 ([#730](https://github.com/MetaMask/controllers/pull/730))
+
+### Fixed
+- Catch errors thrown from `ethereumjs-util` methods in `KeyringController` ([#739](https://github.com/MetaMask/controllers/pull/739))
```

Comment:
This doesn't seem like a great description :thinking: The change is that the error message is different. There is nothing "caught" that was previously "uncaught", as we're re-throwing in the `catch` block.

Something like this might be better:
```suggestion
- Improve error message when attempting to import an invalid private key ([#739](https://github.com/MetaMask/controllers/pull/739))
```